### PR TITLE
Added support for extensibleattributedef

### PR DESCRIPTION
**Pull request by JkhatriInfobox**

### CODE REVIEW

**Comment by JkhatriInfobox** [comment]
@mattclay Please review and merge PR.

**Comment by JkhatriInfobox** [comment]
Hi @mattclay,

I hope you’re well. I’m following up on my pull request. Could you please provide an update or let me know if there’s anything else needed from my end to proceed?

File: flaskapp.py [hunk 1]
```diff
@@ -105,6 +105,7 @@ class NetworkView(object):
     rules = []
     use_logic_filter_rules = False
     logic_filter_rules = []
+    flags = 'STRING'
```
Comment (mattclay) [inline flaskapp.py:108]: Is this attribute used anywhere?

Comment (JkhatriInfobox) [reply flaskapp.py:108]: Yes this field is supported for eadefinition object with pr https://github.com/infobloxopen/infoblox-ansible/pull/235, This field contains extensible attribute flags.

Comment (JkhatriInfobox) [reply flaskapp.py:108]: Waiting for this PR to merge, So CI will pass for integration test added in https://github.com/infobloxopen/infoblox-ansible/pull/246

**Pull request merged**